src/main/java/com/booster/server/VocabularyEntryService.java

```diff
@@ -41,4 +41,17 @@ public void updateLastSeenAt(UpdateLastSeenAtInput input) {
         vocabularyEntryRepository.saveAll(vocabularyEntries);
     }
 
+    public void updateCorrectAnswersCount(UpdateCorrectAnswersCount input) {
+        Long id = input.getId();
+        VocabularyEntryEntity vocabularyEntry = vocabularyEntryRepository.findById(id)
+                .orElseThrow(() -> new RuntimeException("Vocabulary entry with id=%s not found".formatted(id)));
```

Comment:
Let's follow the concise logging approach and change it to `Vocabulary entry not found [id=%s]`?